clash_lib/Cargo.toml

```diff
@@ -15,7 +15,7 @@ tokio = { version = "1", features = ["full"] }
 tokio-util = { version = "0.7", features = ["net", "codec", "io", "compat"] }
 thiserror = "1.0"
 async-trait = "0.1"
-anyhow = "1.0"
+anyhow = { version = "0.6", package = "eyre" }
```

Comment:
can we use the latest version? and is`eyre` useful?

Reply:
this change the whole `anyhow` into `eyre` (https://docs.rs/eyre/latest/eyre/) which is a fork a anyhow. It has more features, in this pr, i use it just for printing more backtrace by default.

Reply:
And with eyre we could get an fancy error output :)
https://github.com/eyre-rs/color-eyre

Reply:
i don't think we need this, at least not in this PR

Reply:
Okay, another day job